### ISSUE TITLE
Company theme not working

### ISSUE THREAD

**Issue by rodakovia** [iomad/iomad#451]
I'm trying to set a theme by company, the IOMAD module makes the change in database but it doesn't work. Any possible solution for this? Thanks in advance.

PD: Sorry about my english :(

**Comment by thepurpleblob** [comment]
You don't say which version of Iomad you are using, but make sure you have the latest 3.1. You would be better using 'Iomadbootstrap' theme which is now the default and fully supported.

**Comment by rodakovia** [comment]
I download yesterday from here a zip (Iomad 31 stable). I tried to set 'iomadbootstrap' as default and then some other themes in other companies, but it didn't work either. :(
The company theme is always the theme set as default in moodle config

**Comment by thepurpleblob** [comment]
Can you give me more info about what "doesn't work" mean? What happens?

Do you mean the config.php file when you say 'moodle config'?

**Comment by rodakovia** [comment]
Sorry ;) I use theme "A" in config moodle. Then, in IOMAD settings I select another theme "B" into company settings, it saves the info in database (mdl_company) but when i login with a user of this company the theme stills being the theme "A"

**Comment by thepurpleblob** [comment]
Ok... I see, will check.

**Comment by rodakovia** [comment]
Moodle config, I mean I set the theme "A" in theme selector in appearance --> theme selector, many thans for all your help and sorry again about my english

**Comment by turf212** [comment]
Have you turned off allow user themes?  Iomad uses this to give users their company theme.

**Comment by rodakovia** [comment]
Yes, I tried both options  :(

**Comment by turf212** [comment]
If you select the clean theme and edit the other user's profile, are you able to change the theme for the other user directly?  Does the theme change for that user when you do.  If you are logged in as that user you need to log out and back in for it to pick up the changes.

The above is standard Moodle operation and nothing to do with Iomad.  Hopefully it will allow you to investigate further.

**Comment by thepurpleblob** [comment]
Works for me. I did this...

- I created a completely new/clean site with the latest iomad31 code
- I went to the Iomad Dashboard and was prompted to create the 1st company which I did. I selected 'Clean' as the theme in the company's appearance section
- Then from the UserManagement tab, I created a new user for that company (all default settings)
- I logged in as that user and the theme was displayed as 'Clean' not 'IomadBootstrap'

I didn't change any settings from the default.
